app/models/music.rb

```diff
@@ -0,0 +1,67 @@
+# Includes name of the music item, linking to appropriate Spotify page
+# - Includes type of the music item (artist, album, track, or playlist)
+# - Includes photo/album cover of the music item
+# - Includes embedded Spotify player if appropriate (optional)
+
+require 'rspotify'
+
+class Music < ActiveRecord::Base
+  # receives and models data retrieved from the Spotify API
+  BASE_URL = "https://api.spotify.com"
+  attr_reader :artist, :album, :track, :music, :name, :img, :url
+
```

Comment:
Since you removed a bunch of these attributes from the initialize method (via commenting them out), you should remove them from here as well.